### DISCUSSION TITLE
Is there a way to save the entire server at once?

### DISCUSSION THREAD

**Discussion by greathiago** [Tyrrrz/DiscordChatExporter#1147] [General]
I wanted to press a button and save the server as a whole
Or server folders at once

If not, perhaps a right-click implementation would be good

I can make it and send a PR, I just need to know where I would need to implement the function and which function I could base this new one

**Comment by CanePlayz** [comment]
You can select multiple (or all) channels of a server by using Shift + Left Click or Control + Left Click, just as you would select files in File Explorer.

**Reply by greathiago** [reply]
This still takes a bit longer than two clicks, because at minimum, you have to open one category and scroll to the end of it. It's as if in a Modal component you deny an implementation of "click outside to close" because it has the X button

That's why I don't think it's very productive to discuss this without a developer from the project itself (or the owner) to analyze the cost/benefit of this implementation.

**Reply by 96-LB** [reply]
@CanePlayz is a contributor to the project and one of the most active members of the support group for this project. Please be civil.

**Reply by CanePlayz** [reply]
> you have to open one channel and scroll to the end of it.

I'm starting to think you mean an implementation into Discord directly instead of an upgrade to the GUI app of DCE?

**Reply by greathiago** [reply]
Sorry, I used the wrong word, I meant category instead of channel. Edited
My suggestion is in the GUI app

**Reply by CanePlayz** [reply]
Okay, that makes sense. If you're interested in efficiency and automation, I'd recommend using the CLI version.
